src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+
+
+export class TodoForm extends React.Component{
+
+    constructor() {
+        super();
+        this.state = {
+          task: '',
+          
+        };
+      } 
+
+
+
+
+    handleInput = (event) =>{
+        this.setState({ [event.target.name]:event.target.value });
+        localStorage.setItem(event.target.name , event.target.value )
```

Comment:
Right now, when you submit an item to the list of todos, the input field doesn't reset. Instead it shows the previous entry. You can reset by resetting the state of task back to ""